plugins/woocommerce-blocks/docs/block-references/block-references.md

```diff
@@ -0,0 +1,1187 @@
+# Woo Blocks Reference
+
+This page lists the Woo blocks included in the package.
```

Comment:
I think it's worth mentioning the list may be incomplete due to some (usually older) blocks being registered without `block.json` and hence omited by script.